src/main/webapp/web.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
```

Comment:
I don't think a lot of the HTML files in this web.xml are named this way yet. We should't be adding this in this PR. Instead, we should just remove references to files that dont exist or are incorrectly named, and then add that code back in during the relevant PRs.

Reply:
discussed offline!